deployments/external-user-management/helmfile.yaml

```diff
@@ -392,6 +391,9 @@ releases:
           web:
             persistence:
               enabled: true
+            config:
+              oidc:
+                webClientID: web
```

Comment:
@wkloucek IMHO we need to add this here. It's the default anyway. I wonder why the default was part of the helmfile before.

Reply:
Just discussed: it's not needed, but left for documentation purposes.